apps/client/utils/common/lastItem.ts

```diff
@@ -0,0 +1,5 @@
+export function lastItem<T>(arr: Util.NonEmptyArray<T>): T;
+export function lastItem<T>(arr: T[]): T | undefined;
+export function lastItem<T>(arr: T[]): T | undefined {
+  return arr[arr.length - 1];
+}
```

Comment:
```suggestion
export function lastItem<T>(arr: Util.NonEmptyArray<T>): T;
export function lastItem<T>(arr: T[]): T | undefined {
  return arr[arr.length - 1];
}```

이거랑 다른 점이 있는 건가요? 알려주십쇼!

Reply:
<img width="1380" alt="image" src="https://user-images.githubusercontent.com/69495129/213918613-a4f4a89b-a6a3-4b97-a38f-7ddf640b0b80.png">

제안해주신 코드처럼 작성하면, 타입추론이 export function lastItem으로만 추론되고 실제 구현체는 타입추론이 안됩니다!😁

Reply:
오오 그렇네요 감사합니다!